tasks/destroy.py

```diff
@@ -11,6 +11,7 @@
 
 def destroy(
     ctx: Context,
+    *,
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
TIL, should it be at the end instead?

Reply:
No, this is how we enforce callers to use named params and not positional ones 🙂